libvirt/kcli_cluster.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+#
+# Copyright Confidential Containers Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# Manage Kubernetes clusters with kcli.
+#
+set -o errexit
+set -o nounset
+set -o pipefail
+
+CLUSTER_DISK_SIZE="${CLUSTER_DISK_SIZE:-20}"
+CLUSTER_MASTERS="${CLUSTER_MASTERS:-1}"
+CLUSTER_NAME="${CLUSTER_NAME:-peer-pods}"
+CLUSTER_IMAGE="${CLUSTER_IMAGE:-ubuntu2004}"
+CLUSTER_WORKERS="${CLUSTER_WORKERS:-1}"
+LIBVIRT_NETWORK="${LIBVIRT_NETWORK:-default}"
+LIBVIRT_POOL="${LIBVIRT_POOL:-default}"
+
+# Wait until the command return true.
+#
+# Parameters:
+#   $1 - wait time.
+#   $2 - sleep time.
+#   $3 - the command to run.
+#
+wait_for_process() {
+	local wait_time="$1"
+	local sleep_time="$2"
+	local cmd="$3"
+
+	while ! eval "$cmd" && [ "$wait_time" -gt 0 ]; do
+		sleep "$sleep_time"
+		wait_time=$((wait_time-"$sleep_time"))
+	done
+
+	[ "$wait_time" -ge 0 ]
+}
+
+# Create the cluster.
+#
+create () {
+	kcli create kube generic \
+		-P domain="kata.com" \
+		-P pool="$LIBVIRT_POOL" \
+		-P masters="$CLUSTER_MASTERS" \
```

Comment:
In my version of kcli (version: 99.0 commit: 8bb9ccd 2023/01/19 Available Updates: False), I got:
```
# ./kcli_cluster.sh create
parameters that contain master word need to be replaced with ctlplane. Found masters
```
so had to change this to `-P ctlplanes="$CLUSTER_MASTERS" \` Does that work on your version and if so can we also update the name of `CLUSTER_MASTERS` to be more inclusive. Thanks!

Reply:
hi Steve!

I'm using a slightly older version of kcli: version: 99.0 commit: 8eeddd2 2022/10/03 Available Updates: N/A

The `ctlplanes` parameter worked out with that version. I will update the script to also remove `CLUSTER_MASTERS`

Reply:
done

Reply:
Thanks - the changes look good